src/stubs/win32/opamWindows.c

```diff
@@ -549,24 +551,29 @@ CAMLprim value OPAMW_process_putenv(value pid, value key, value val)
     caml_failwith(result);
 }
 
-/*
- * Somewhat against my better judgement, wrap SHGetFolderPath rather than
- * SHGetKnownFolderPath to maintain XP compatibility. OPAM already requires
- * Windows Vista+ because of GetCurrentConsoleFontEx, but there may be a
- * workaround for that for XP lusers.
- */
-CAMLprim value OPAMW_SHGetFolderPath(value nFolder, value dwFlags)
+value OPAMW_SHGetKnownFolderPath(REFKNOWNFOLDERID rfid)
```

Comment:
```suggestion
static value OPAMW_SHGetKnownFolderPath(REFKNOWNFOLDERID rfid)
```
(the function name could then be de-prefixed as well, if you want, e.g. `get_current_known_folder_path` or some such)